### DISCUSSION TITLE
Katex support for gatsby-starter-minimal-blog

### DISCUSSION THREAD

**Discussion by smejak** [LekoArts/gatsby-themes#758] [Questions]
I'd like to render math equations in the gatsby-starter-minimal-blog, but when I followed the guide on adding katex to gatsby https://www.gatsbyjs.com/plugins/gatsby-remark-katex/ it didn't work. Does anyone have any recommendations on how to fix this?

**Comment by George9718** [comment]
Hey,

I'm trying to do the same - did you ever find a solution?

**Comment by FarhadG** [comment]
Tried doing the same but no luck. I'm considering bringing the node_modules dependencies into the main project to modify the source code and build from there. Doesn't seem like it's the right approach though
